pkg/prometheus/rules.go

```diff
@@ -0,0 +1,54 @@
+package prometheus
+
+import (
+	monv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	monitoringclient "github.com/coreos/prometheus-operator/pkg/client/versioned"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var ruleLog = logf.Log.WithName("common_prometheusRule")
+
+func newMonitoringClient() (*monitoringclient.Clientset, error) {
```

Comment:
make function name a pascal case

Reply:
This is not a public function. Pascal Case would make it public.